test/functionalTest/harnessFunctions.sh

```diff
@@ -450,7 +450,7 @@ function accumulatorStart()
 
   accumulatorStop $port
 
-  accumulator-server.py $port /notify $bindIp 2> /tmp/accumulator_$port &
+  accumulator-server.py $port /notify $bindIp > /tmp/accumulator_${port}_stdout 2> /tmp/accumulator_$port &
```

Comment:
Sure, why not.
I didn't change this just to not change already existing names.
But, the idea is good. Let's do it ! ;-)